lists/id/id-djp.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Director General of Taxes",
+    "local": "Direktur Jenderal Pajak"
+  },
+  "url": "http://ceknpwp.com/",
+  "description": {
+    "en": "The Directorate General of Taxes is Indonesia's tax office.\n\nThe Tax Identification Number (TIN) is known in Indonesia as Nomor Pokok Wajib Pajak (NPWP) and is issued by the Directorate General of Taxes to individuals and entities. "
+  },
+  "coverage": [
+    "ID"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "ID-DJP",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": "A Tax Identifier Lookup is available at http://ceknpwp.com/\n\nIt is necessary to know an organisation's NPWP in order to perform the lookup. All non-numeric characters must be removed from the NPWP. The lookup may require one or more captchas to be completed before showing the result. There is no sign up. Upon successfully completing the lookup only the organisation's name is shown.",
+    "publicDatabase": "http://ceknpwp.com/",
+    "guidanceOnLocatingIds": "You need to know the identifier in advance. Lookups are from identifier to company only.",
+    "exampleIdentifiers": "010117771077000, 312666837412000",
+    "languages": [
+      "ID"
+    ]
+  },
+  "data": {
+    "availability": [
+      "data_not_available"
+    ],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "no_license",

```

Comment:
I think the license status should be blank, given that the rest of the data object is blank

Reply:
Agreede. 'licenseStatus' edited to blank